main.js

```diff
@@ -380,8 +380,7 @@ adapter.on('ready', () => {
 }*/
 
 function initNodes(nodes) {
-    adapter.log.info('initialize ' + nodes.length + ' nodes');
-    adapter.log.silly('Received NODES: ' + JSON.stringify(nodes));
```

Comment:
ehy you removed the silly logging here? It's good for debugging

Reply:
Readded it again now. Seemed broken at first and gave me a type error regarding circular references, but is working now.

Reply:
I will change that back because the button in admin is not that relevant ... but setStates shuld work and it is needed for legacy support